docs/guides/file-transfer.md

```diff
@@ -27,10 +27,11 @@ possible formats this path can take:
 * `<container-type>` is the container type
     * On simulators, common values are `app`, `data`, `groups`, but a custom one can also be provided
     * On real devices, the only accepted value is `documents`. All others are treated as Format 2
-        * In `xcuitest-driver` versions prior to version `v8.3.0` the following limitation applies:
-        This value can only be specified for apps that have the `UIFileSharingEnabled` flag set to
+        * This value can only be specified for apps that have the `UIFileSharingEnabled` flag set to
           `true`. You can use the [`mobile: listApps`](../reference/execute-methods.md#mobile-listapps)
           extension to identify such apps.
+        * `skipDocumentsContainerCheckInFileManagement` [Settings API](https://appium.io/docs/en/latest/guides/settings/) helps you to avoid this limitation on certain cases..
```

Comment:
By assigning the `skipDocumentsContainerCheck` setting to `true` you may skip the above limitation for certain apps.